DEPS

```diff
@@ -1101,5 +1117,16 @@ hooks = [
       'third_party/impeller-cmake-example',
       '--setup',
     ]
+  },
+  {
```

Comment:
How big are these system images? Most Flutter engine developers aren't developing/testing for Fuchsia, so if these are bigger than a few MB, this download should be gated behind a gclient custom var or env variable that is set on CI.

Reply:
It's fairly large. We discussed if it was a common practice to have a `download_fuchsia_deps` to consider both `host_os` and `target_os`, but haven't found a similar approach for other platforms. https://github.com/flutter/engine/pull/49650#discussion_r1449235275